README.md

```diff
@@ -276,17 +270,17 @@ let credential = await navigator.credentials.get({
     providers : [
       {
         origin: "https://login.idp.net",
-        loginURL: "https://bounce.example.com/?u=https://login.idp.net/login.html",
+        loginURL: "https://bounce.example.com/?u=https://login.idp.net/login.html?r=https://rp.net/",
         loginTarget: "redirect",
       },
     ]
   }
 });
 ```
 
-This example shows the use perfect for a "Log in with Foo" button (use case #1, and use case #2), where one identity provider is presented, and if the user has not already logged in, they may be redirected to that provider's login page. This redirect behavior is only permitted when there is only one provider in the list. A provider with `loginURL` field indicates that this is the expected mode. If `loginURL` is present, but `origin` is not, its value can be inferred as the origin of the link.
+This example shows the use perfect for a "Log in with Foo" button, where one identity provider is presented, and if the user has not already logged in, they may be redirected to that provider's login page. This redirect behavior is only permitted when there is only one provider in the list. A provider with `loginURL` field indicates that this is the expected mode. If `loginURL` is present, but `origin` is not, its value can be inferred as the origin of the link.
```

Comment:
What's "the use perfect"?

Reply:
it could be rewritten as "the use of the API that is perfect". "use" as in the noun meaning occasion or need. Agreed that this could be reworded

Reply:
```suggestion
This example shows a scenario that is perfect for a "Log in with Foo" button, where one identity provider is presented, and if the user has not already logged in, they may be redirected to that provider's login page. This redirect behavior is only permitted when there is only one provider in the list. A provider with `loginURL` field indicates that this is the expected mode. If `loginURL` is present, but `origin` is not, its value can be inferred as the origin of the link.
```